src/cpp/src/whisper/models/with_past_decoder.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "decoder.hpp"
+#include "openvino/runtime/core.hpp"
+
+namespace ov::genai {
+
+class WhisperWithPastDecoder : public WhisperDecoder {
+public:
+    explicit WhisperWithPastDecoder(const std::filesystem::path& models_path,
```

Comment:
`explicit` is not required as ctor has more than 1 mandatory argument

Reply:
Thanks, fixed